### ISSUE TITLE
[FEAT]: Voice & Speech Autoplay

### ISSUE THREAD

**Issue by mallory303** [Mintplex-Labs/anything-llm#2237]
### What would you like to see?

There should be a flag for "auto playback" to play the audio stream / file when it's completed
More settings for elevenlabs: selectable model

**Comment by F-U-B-AR** [comment]
Agreed, and an option to display the text as the LLM says it, or display all text only after the voice has finished speaking. 

Also more local options! xtts, couqui tts etc

**Comment by AIbottesting** [comment]
I long for the days when the voice feature was first released it would show your words that it picked up and then you could press the send button when you were finished speaking. For quite some time now it just cuts me off after about 4 words and sends it automatically. It sucks. I wish there was a settings toggle switch were people could at least choose which way they like it.
